djoser/social/token/jwt.py

```diff
@@ -1,9 +1,11 @@
 class TokenStrategy:
     @classmethod
     def obtain(cls, user):
-        from rest_framework_jwt.settings import api_settings
-        payload = api_settings.JWT_PAYLOAD_HANDLER(user)
+        from rest_framework_simplejwt.tokens import RefreshToken
```

Comment:
Can this be changed so that it stays backward-compatible? For example someone may use `JWT_PAYLOAD_HANDLER` - what then? If not, a deprecation route must be taken.